src/components/stats/garden/composter-upgrades.svelte

```diff
@@ -0,0 +1,124 @@
+<script lang="ts">
+	import {
+		getCropDisplayName,
+		getComposterUpgradeDisplayName,
+		getSpecialCropDisplayName,
+		Crop,
+	} from 'farming-weight';
+	import type { components } from '$lib/api/api';
+	import { PROPER_CROP_TO_IMG, SPECIAL_CROP_TO_IMG } from '$lib/constants/crops';
+	import { API_COMPOSTER_UPGRADE_TO_UPGRADE, COMPOSTER_UPGRADE_TO_IMG } from '$lib/constants/composter';
+	import * as Popover from '$ui/popover';
+	import { getComposterUpgradeCost, getEnchantedCropCollectionAmount } from '$lib/calc/garden';
+
+	interface Props {
+		garden?: components['schemas']['GardenDto'];
+	}
+
+	let { garden = undefined }: Props = $props();
+
+	const maxLevel = 25;
+
+	let composterUpgrades = $derived(
+		garden?.composter?.upgrades || {
+			speed: 0,
+			multi_drop: 0,
+			fuel_cap: 0,
+			organic_matter_cap: 0,
+			cost_reduction: 0,
+		}
+	);
+
+	let upgrades = $derived(
```

Comment:
All of this max level/upgrades calculation should probably be moved to the `farming-weight` package PR, as I imagine it'll also be used on the bot for a display there